md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/util/Dao2TableUtil.java

```diff
@@ -73,11 +73,18 @@ public static Process buildJPAProcess(Integer processTypeId, String name, String
         ProcessType daoProcessType = new ProcessType();
         daoProcessType.setProcessTypeId(processTypeId);
         daoProcess.setProcessType(daoProcessType);
+        LOGGER.info("workflow type id is "+workflowTypeId);
         if (workflowTypeId != null) {
             com.wipro.ats.bdre.md.dao.jpa.WorkflowType daoWorkflowType = new com.wipro.ats.bdre.md.dao.jpa.WorkflowType();
             daoWorkflowType.setWorkflowId(workflowTypeId);
             daoProcess.setWorkflowType(daoWorkflowType);
         }
+        else
+        {
+            com.wipro.ats.bdre.md.dao.jpa.WorkflowType daoWorkflowType = new com.wipro.ats.bdre.md.dao.jpa.WorkflowType();
+            daoWorkflowType.setWorkflowId(1);
```

Comment:
@Prem1993 please fetch the obj of workflowtype and then set the whole object instead of setting the id manually